### PR TITLE
StorefrontContent: mention page/block scope

### PR DESCRIPTION
**Pull request by real34**
Our existing example will generate block scoped X-Ray.

This PR adds a paragraph to show how to do a page scoped content.

:arrow_right: [Preview](https://deploy-preview-678--heuristic-almeida-1a1f35.netlify.app/docs/2.x/magic-button/x-ray#add-storefrontcontent--in-your-react-components)

### CODE REVIEW

File: versioned_docs/version-2.x/magic-button/x-ray.mdx [hunk 1]
```diff
@@ -119,6 +119,14 @@ const MyCustomType = ({ aMyCustomType }) => {
 }
 ```
 
+By default, the X-Ray view will be scoped to the **block**.
```
Comment (dpobel) [inline versioned_docs/version-2.x/magic-button/x-ray.mdx:122]: maybe: 

```suggestion
By default, the X-Ray view will be scoped as a **block**.
```

**Review by dpobel — APPROVED** [review]

**Review by paul-vd — CHANGES_REQUESTED** [review]
Just by reading this, it's hard to understand what a `block` and what a `page` is, it could be good to add a comparison screenshot in a table to serve as an example.

|  Block | Page |
| ------ | ----- |
| SCREENSHOT HERE | SCREENSHOT HERE |

File: versioned_docs/version-2.x/magic-button/x-ray.mdx (hunk 1 above)
Comment (real34) [reply versioned_docs/version-2.x/magic-button/x-ray.mdx]: I amended the commit with this suggestion.

**Comment by real34** [comment]
> Just by reading this, it's hard to understand what a `block` or `page` represents, it could be good to add a comparison screenshot in a table to serve as an example.
> 
> Block	Page
> SCREENSHOT HERE	SCREENSHOT HERE

I added a single annotated screenshot, in order to show the difference in a PLP context featuring both scopes on the same page.

**Review by paul-vd — APPROVED** [review]

**Pull request merged**